scientist-softserv/harvard-curiosity
# High priority design refinements

**High priority design refinements**
* * *

**JIRA Ticket**:
* https://github.com/harvard-lts/CURIOSity/issues/248

# What does this Pull Request do?
Resolves a most tasks on the "high priority" ticket, related to:
* showing titles in masonry view for items without thumbnail images
* search box style issues in safari and chrome
* restyling exhibit tags on item detail pages to look like links
* normalizing styles of each tag box in Configuration --> Tags tab
* update text on homepage "explore more collections" card

# How should this be tested?
* Find an exhibit that has a few items without thumbnail images (I used contagion) and navigate to the search results page in masonry view (6th page, showing 96 per page, should include a few without thumbnails)
   * If items have thumbnail images, you will need to hover over image to see the title 
   * If items don't have thumbnail images, title will always appear with a light grey background (also flushed to the top and taking up 70px of height, which is the height needed if a title is 3 lines long)
* Search box style was a little off in Safari and Chrome
   * Navigate to an exhibit in Safari - the right-facing arrow in the searchbox should be inline with the word "search" (was previously wrapping to a second line and appearing below the word)
   * Navigate to an exhibit in Chrome that has fielded searching enabled - dropdown box and "Everything" text no longer runs into the carrots
* Navigate to an item detail page that includes tags (though I found it easiest to just create 2-3 tags for a few items instead of searching)
   * Exhibit tags now appear styled as links on item detail pages (previously small buttons, will still look this way in the edit view)
* Navigate to the tags page under the admin dashboard (you should see the tags that you created here)
   * Each row of tag information should look consistent: titles appear as they were typed (i.e. case sensitive), edit links have a smaller font size, and delete buttons are all the same size
* On homepage, inspect the "Explore More Collections" card
   * The link title should now read "Explore More Collections" to match the heading text (previously said "Explore all collections")


# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests?
- integration tests?

# Interested parties
@phil-plencner-hl @dl-maura
